_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -841,26 +821,6 @@ public static function verify_site( $request ) {
 		}
 	}
 
-	/**
-	 * Handles verification that a site is registered
-	 *
-	 * @since 5.4.0
-	 *
-	 * @param WP_REST_Request $request The request sent to the WP REST API.
-	 *
-	 * @return array|wp-error
-	 */
-	 public static function remote_authorize( $request ) {
```

Comment:
We're removing 3 methods from this class:
* `remove_authorize`
* `jetpack_connection_status`
* `get_connection_plugins`

Do you think it would be safer to deprecate this methods instead of removing them?

Reply:
Good point.
I added back the `remote_authorize()` and `jetpack_connection_status()`. They don't seem to be used, but it's a good idea to deprecate them for now.
I added `get_connection_plugins()` myself two months ago, and it's hard to imagine anybody using it. So I think we're safe removing it entirely.